src/tests/class_tests/openms/source/InclusionExclusionList_test.cpp

```diff
@@ -45,6 +45,9 @@
 using namespace OpenMS;
 using namespace std;
 
+#pragma clang diagnostic push
+#pragma clang diagnostic ignored "-Wshadow"
```

Comment:
no, but for the test I actually dont really care - however, it pollutes the warnings in cdash